test/AllocatorBuffer.t.sol

```diff
@@ -36,12 +36,15 @@ contract AllocatorBufferTest is DssTest {
         assertEq(gem.allowance(address(buffer), address(0xBEEF)), 10);
     }
 
-    function testDeposit() public {
+    function testDepositWithdraw() public {
         assertEq(gem.balanceOf(address(this)),   gem.totalSupply());
         assertEq(gem.balanceOf(address(buffer)), 0);
         gem.approve(address(buffer), 10);
-        buffer.deposit(address(gem), 10, address(0));
+        buffer.deposit(bytes32(0), address(gem), 10);
```

Comment:
can test the events on `withdraw` and `deposit`.

Reply:
same